core/services/workflows/metering/metering.go

```diff
@@ -712,7 +788,7 @@ func (s *Reports) Get(workflowExecutionID string) (*Report, bool) {
 }
 
 // Start creates a new report and inserts it under the specified workflowExecutionID.
-func (s *Reports) Start(ctx context.Context, workflowExecutionID string) (*Report, error) {
+func (s *Reports) Start(ctx context.Context, workflowExecutionID string, engineVersion string) (*Report, error) {
```

Comment:
why pass a static string on Start instead of once in the constructor?

Reply:
Moved this value to the `NewReports` constructor function.